tests/keras/test_keras_model_export.py

```diff
@@ -44,42 +44,43 @@ def predicted(model, data):
 
 def test_model_save_load(model, data, predicted):
```

Comment:
nit: here we can use pytest's `tmpdir` fixture (i.e. just pass `tmpdir` to the test function and it'll automatically create/cleanup a tempdir for us)